README.md

```diff
@@ -1,5 +1,11 @@
 # software-templates
-This repository is intended to house Backstage Software Templates for creating new services, as well as a variety of documentation templates. 
+
+This repository is a collection of software and documentation templates for the Backstage community. The templates are organized into two folders: `scaffolder-templates` for Backstage Software Templates and `documentation-templates` for general documentation templates. Contributions are welcome for new and existing templates!
+
+## scaffolder-templates
+
+[Backstage Software Templates](https://backstage.io/docs/features/software-templates/) are used to create new software components through Backstage. These templates are intended as a starting point to build on for different use cases. You may also use them as provided, but we recommend pointing your Backstage app to a forked copy because the templates aren't versioned at this time.
```

Comment:
```suggestion
[Backstage Software Templates](https://backstage.io/docs/features/software-templates/) are used to create new software components through Backstage. These templates are intended as a starting point to build on for different use cases. You may also use them as provided, but we recommend pointing your Backstage app to a forked copy because the templates aren't versioned at this time, and as such breaking changes can happen without warning.
```

Wonder if we could make this a little bit more strongly worded with the above change?